src/StackAdmin/Azs.Commerce.Admin/Module/Azs.Commerce.Admin/0.1.0/Azs.Commerce.Admin.Resources.psd1

```diff
@@ -0,0 +1,46 @@
+<#
```

Comment:
What is this?  Please remove

Reply:
This is AutoGenerated from PSSwagger.  From my understanding they use this for displaying errors.